source/extensions/stat_sinks/open_telemetry/open_telemetry_impl.h

```diff
@@ -42,13 +42,17 @@ class OtlpOptions {
   bool emitTagsAsAttributes() { return emit_tags_as_attributes_; }
   bool useTagExtractedName() { return use_tag_extracted_name_; }
   const std::string& statPrefix() { return stat_prefix_; }
+  const absl::flat_hash_map<std::string, std::string>& resource_attributes() {
+    return resource_attributes_;
+  }
 
 private:
   const bool report_counters_as_deltas_;
   const bool report_histograms_as_deltas_;
   const bool emit_tags_as_attributes_;
   const bool use_tag_extracted_name_;
   const std::string stat_prefix_;
+  absl::flat_hash_map<std::string, std::string> resource_attributes_;
```

Comment:
nit: I don't think this is used as map anywhere, or that the elements are ever accessed individually. Consider converting it to the type that can be copied into `resource_metrics->mutable_resource()->mutable_attributes()` in open_telemetry_impl.cc.

Reply:
DONE